### PR TITLE
Update response 2

### PR DESCRIPTION
**Pull request by ros3bud**
@cgrayson - updated to reflect the suggestions you made. 

I apologize for the new branch, but my old one had bugs which I suspect were caused by my IDE.

### CODE REVIEW

File: spec/outbound-spec.coffee [hunk 1]
```diff
@@ -286,27 +286,92 @@ describe 'Outbound', ->
...
+        {
+          "id": 923934,
+          "first_name": "Lynda",
+          "last_name": "Test",
+          "email_address": "lyndaw2@activeprospect.com",
+          "registration_date": "2015-04-21T11:55:01-06:00"
+        }
+        """
+      status = 201
+      res = outbound.response(@vars, @req, response(body, status))
```
Comment (cgrayson) [inline spec/outbound-spec.coffee]: Missed eliminating `status` here and in the next test.

File: src/outbound.coffee [hunk 2]
```diff
@@ -115,33 +115,26 @@ request.variables = ->
...
     else
-      # The response status is NOT in the 200 range. Treat this as an error.
-      outcome: 'error'
-      reason: res.body
-
+      reason: errorReason
+      outcome: 'error'    
+  
   # Return the data to append
   appendData
```
Comment (cgrayson) [inline src/outbound.coffee]: The local var `appendData` can now actually be done away with entirely. I.e., take out lines 120, 135 & 136, and outdent the if-else block; each condition will return the outcome/reason object.

**Comment by cgrayson** [comment]
Otherwise, LGTM

File: src/outbound.coffee (hunk 2 above)
Comment (ros3bud) [inline src/outbound.coffee]: Updates made - thanks for the feedback!

**Pull request merged**